services/121-service/package.json

```diff
@@ -70,6 +70,7 @@
     "tsconfig-paths": "^4.2.0",
     "twilio": "^5.0.4",
     "typeorm": "^0.3.17",
+    "typescript": "^5.5.0-beta",
```

Comment:
LOL. Why did it work before in the first place? :D

`ts-node` was probably 'fixing it for us'... or something..